skimage/filters/rank/generic.py

```diff
@@ -51,7 +51,7 @@
 import functools
 import numpy as np
 from scipy import ndimage as ndi
-from ... import img_as_ubyte
+from ...util import img_as_ubyte
```

Comment:
This is very much a fair point. I'll let @scikit-image/core comment before I make that change, but I am :+1: on this suggestion.

Reply:
Do you mind expanding on the issue here? What is being proposed?

Reply:
@stefanv I think @hmaarrfk is making a distinction between precision loss warnings stemming from IO and user calls to `img_as_ubyte`, which should be silenced, and precision loss warnings because rank filters convert your images to uint8, and now do so completely silently.

Having said that I would like to punt on this issue, as I don't consider it that critical or urgent. (See main conversation thread on the PR.)

Reply:
>convert your images to uint8, and now do so completely silently

I think that happens in multiple other places also. 👍 to issue the warning.

Reply:
Done.